app/assets/javascripts/app/radio-btn.js

```diff
@@ -0,0 +1,39 @@
+import 'classlist.js';
+
+function clearRadioBtn(name) {
+  const radioGroup = document.querySelectorAll(`input[name='${name}']`);
+
+  for (let i = 0; i < radioGroup.length; i++) {
+    radioGroup[i].parentNode.parentNode.classList.remove('bg-light-blue');
+  }
+}
+
+function radioBtn() {
+  const radios = document.querySelectorAll('.radio-btn input[type=radio]');
+
+  if (radios) {
+    for (let i = 0; i < radios.length; i++) {
+      const radio = radios[i];
+      const label = radio.parentNode.parentNode;
+      const name = radio.getAttribute('name');
+
+      if (radio.checked) label.classList.add('bg-light-blue');
+
+      radio.addEventListener('change', function() {
+        clearRadioBtn(name);
```

Comment:
I think by updating this to the `.forEach` syntax that resolves the scope issue I was running into before.  I need to get out of the habit of using the `for` loop, thanks for the pointers.